gaas-ui/package.json

```diff
@@ -14,81 +14,89 @@
     "@testing-library/user-event": "^12.5.0",
-    "@types/node": "^14.14.10",
-    "@types/react": "^16.14.2",
-    "@types/react-dom": "^16.9.10",
-    "@types/validator": "^13.6.3",
-    "amazon-cognito-identity-js": "^4.5.5",
-    "axios": "^0.21.1",
+    "@types/node": "^16.11.6",
+    "@types/react": "^17.0.33",
+    "@types/react-dom": "^16.9.14",
+    "@types/validator": "^13.6.5",
+    "amazon-cognito-identity-js": "^4.6.3",
+    "axios": "^0.21.4",
     "body-parser": "^1.19.0",
+    "core-js": "^3.19.0",
     "cors": "^2.8.5",
+    "d3-array": "^3.1.1",
     "env-cmd": "^10.1.0",
     "express": "^4.17.1",
-    "http-proxy-middleware": "^1.0.6",
+    "http-proxy-middleware": "^1.3.1",
     "json": "^10.0.0",
     "jsonwebtoken": "^8.5.1",
     "material-ui-dropzone": "^3.5.0",
-    "material-ui-image": "^3.3.0",
+    "material-ui-image": "^3.3.2",
     "npm-check-updates": "^9.1.0",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
     "react-json-view": "^1.19.1",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
+    "react-router": "^5.2.1",
+    "react-router-dom": "^5.3.0",
     "react-scripts": "^4.0.0",
     "react-testing-library": "latest",
     "react-toastify": "^6.1.0",
-    "reaviz": "^10.1.0",
-    "sanitize-html": "^2.4.0",
+    "reaviz": "^10.4.4",
+    "regenerator-runtime": "^0.13.9",
+    "sanitize-html": "^2.5.2",
     "statuses": "^2.0.1",
     "validator": "^13.6.0"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.12.7",
-    "@babel/preset-react": "^7.12.7",
-    "@testing-library/react": "^11.2.2",
-    "@types/body-parser": "^1.19.0",
-    "@types/enzyme": "^3.10.8",
+    "@babel/preset-env": "^7.15.8",
+    "@babel/preset-react": "^7.14.5",
+    "@testing-library/react": "^11.2.7",
+    "@types/body-parser": "^1.19.1",
+    "@types/core-js": "^2.5.5",
+    "@types/enzyme": "^3.10.10",
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/express": "^4.17.9",
-    "@types/jest": "26.0.15",
-    "@types/morgan": "^1.9.2",
-    "@types/react-router-dom": "^5.1.6",
-    "@types/sanitize-html": "^2.3.2",
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.2",
+    "@types/morgan": "^1.9.3",
+    "@types/react-router-dom": "^5.3.2",
+    "@types/regenerator-runtime": "^0.13.1",
+    "@types/sanitize-html": "^2.5.0",
     "@types/statuses": "^2.0.0",
-    "@types/supertest": "^2.0.10",
-    "@types/webpack": "^4.41.25",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
-    "accessibility-checker": "^3.1.6",
-    "axios-mock-adapter": "^1.19.0",
+    "@types/supertest": "^2.0.11",
+    "@types/webpack": "^4.41.31",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "accessibility-checker": "^3.1.11",
+    "axios-mock-adapter": "^1.20.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
+    "babel-jest": "^27.3.1",
```

Comment:
This is causing dependency issues for me locally

Reply:
Wasn't actually needed so has been removed.